servers/rendering/rasterizer_rd/shaders/scene_high_end.glsl

```diff
@@ -728,24 +792,81 @@ void light_process_omni(uint idx, vec3 vertex, vec3 eye_vec, vec3 normal, vec3 a
 			clamp_rect.y += clamp_rect.w;
 
 		} else {
-
 			splane.z = 1.0 - splane.z;
 		}
 
 		splane.xy /= splane.z;
+
+		{
+
+#if 0
```

Comment:
Is this intentionally left out?

Reply:
ah yes, this is from a previous attempt to implement normal bias in this type of light which I wasn' t very convinced about. The one above works better.